heartbeat/heartbeat.reference.yml

```diff
@@ -1218,6 +1218,9 @@ logging.files:
   # never, once, and freely. Default is never.
   #ssl.renegotiation: never
 
+  #metrics.period: 10s
```

Comment:
@urso I have second thoughts if we even want to add this to the config file, meaning should we even recommend users to change it?

Reply:
It's the reference config. I don't really see a problem in configuring the interval. Allows users to reduce number of documents for monitoring if required.